### PR TITLE
Refactor classes

### PR DESCRIPTION
**Pull request by aniasobo**

### CODE REVIEW

File: forest.py [hunk 1]
```diff
@@ -29,6 +54,10 @@ def tokeniser(self):
         else:
             raise Exception("Error 3")
 
+
+
+
+
 # diffrent types of tokens
```
Comment (josephtownshend) [inline forest.py]: still have some comments in the code.

**Review by josephtownshend — COMMENTED** [review]
couple of comments - otherwise looks great

File: forest.py [hunk 2]
```diff
@@ -22,13 +28,31 @@ def tokeniser(self):
                     strstart = restofstr[0:2]
                     middle = restofstr[2:-2]
                     strstop = restofstr[-2:]
-                    return [{"ECHO": echo},
-                            {"STRSTART" : strstart},
-                            {"STRING" : middle},
-                            {"STRSTOP" : strstop}]
+                    return {"ECHO": echo, "STRSTART" : strstart, "STRING" : middle, "STRSTOP" : strstop}
+                 #   return [{"ECHO": echo},
+                 #           {"STRSTART" : strstart},
```
Comment (ajmpawlik) [inline forest.py]: should we remove these comments?

File: forest.py [hunk 3]
```diff
@@ -22,13 +28,31 @@ def tokeniser(self):
...
+        for key in self.tokens:
+            parse_keys.append(key)
+            
+        return parse_keys
+
+    def run_parser(self):
+        return self.parse_dict[0] == self.keytify_tokens()
+    
+    
 # diffrent types of tokens
```
Comment (ajmpawlik) [inline forest.py]: should we remove these comments?

**Pull request merged**